app/graphql/resolvers/freak_update_resolver.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module Resolvers
+  class FreakUpdateResolver < Resolvers::Base
+    argument :id, GraphQL::Types::ID, required: true
+    argument :first_name, String, required: true
+    argument :last_name, String, required: true
+    argument :description, String, required: true
+    argument :email, String, required: true
+    argument :norm_id, GraphQL::Types::ID, required: true
+    argument :role_id, GraphQL::Types::ID, required: true
+    argument :level_id, GraphQL::Types::ID, required: true
+
+    type [Types::FreakType], null: false
+
+    # TODO: Test if norm_id exist
+    # TODO: Test if role_id exist
+    # TODO: Test if level_id exist
+    def resolve(id:, **attrs)
+      norm = Norm.find(attrs[:norm_id])
+      role = Role.find(attrs[:role_id])
+      level = Level.find(attrs[:level_id])
+
+      freak = Freak.find(id)
+
+      return unless freak
+
+      freak.update(attrs.merge(norm: norm, role: role, level: level))
+      freak
```

Comment:
We should also ensure the freak exists before updating, otherwise an error should be returned stating it doesn't exist (similar to how we'd do for an inexistent norm/role/level ID).

For now we can assume the `Freak` exists - so just find & update the freak, but we'll have to update this when we put validation in place.

Reply:
I add a message in TODO for that